### PR TITLE
Created Database Connection

### PR DESCRIPTION
**Pull request by Jake-Andrews-git**
- Revamped the directories with building of model router connections.
- Instantiated routers 
- Created the python app
## Issues
- Some issues with global libraries and sql queries

### CODE REVIEW

File: .idea/dataSources.xml [hunk 1]
```diff
@@ -8,5 +8,12 @@
       <jdbc-url>jdbc:postgresql://localhost:5432/HackCamp</jdbc-url>
       <working-dir>$ProjectFileDir$</working-dir>
     </data-source>
+    <data-source source="LOCAL" name="HackCamp@hc24_12" uuid="e4a7f4db-55ea-44f3-a00f-791ec7c97494">
+      <driver-ref>postgresql</driver-ref>
+      <synchronize>true</synchronize>
+      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
+      <jdbc-url>jdbc:postgresql://hc24_12:443/postgres</jdbc-url>
```
Comment (crazybrain23) [inline .idea/dataSources.xml:15]: Wouldn't this only work on your system unless we host it locally exactly how you did?

File: app/Stuff I don't need kinda/__init__.py
Comment (crazybrain23) [inline app/Stuff I don't need kinda/__init__.py:1]: Why is does this exist?

File: app/Stuff I don't need kinda/dataQueryTest.py
Comment (crazybrain23) [inline app/Stuff I don't need kinda/dataQueryTest.py:1]: Same as above

File: app/Stuff I don't need kinda/databaseConn.py
Comment (crazybrain23) [inline app/Stuff I don't need kinda/databaseConn.py:1]: Same as above.

File: app/Stuff I don't need kinda/prevmain.py
Comment (crazybrain23) [inline app/Stuff I don't need kinda/prevmain.py:1]: Is this temporary? At the very least add a comment saying it's just for testing.

File: app/config.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import os
+
+DATABASE_CONFIG = {
```
Comment (crazybrain23) [inline app/config.py:3]: These will need to be in a secret file somewhere, to prevent secret leaking.

File: app/db/connection.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
+
+
+import psycopg2
+from psycopg2 import pool
+from psycopg2.extras import RealDictCursor
+from contextlib import contextmanager
+
+# Database configuration
+DB_CONFIG = {
```
Comment (crazybrain23) [inline app/db/connection.py:9]: Why is this duplicated (also in `__init__.py`)? Also it should get it from a secret file.

**Review by crazybrain23 — CHANGES_REQUESTED** [review]
Found a few oddities, also there's already a PR for the models #20, so this PR adding them would be redundant.

**Comment by crazybrain23** [comment]
What issues would this close?

File: app/main.py [hunk 4]
```diff
@@ -1,19 +1,24 @@
+
 from fastapi import FastAPI
 import uvicorn
+from app.routers import commits, repositories, branches, files, licenses, languages
 
 app = FastAPI()
 
-def read_root():
-    return  {"Hello": "World"}
```
Comment (not-Boris) [inline app/main.py:7]: :(

**Review by ghost — APPROVED** [review]

**Review by crazybrain23 — APPROVED** [review]

**Pull request merged**